### DISCUSSION TITLE
Undocumented property in SharesReply (peer code 5) messages

### DISCUSSION THREAD

**Discussion by mathiascode** [jpdillingham/Soulseek.NET#567] [General]
I discovered an undocumented property in SharesReply messages while implementing support for browsing locked/private files. It seems to be rather old, since Soulseek NS clients also send it. I have no idea what it's used for, if you do, let me know. As far as I can tell, both Soulseek NS and SoulseekQt sends a value of 0.

https://github.com/Nicotine-Plus/nicotine-plus/commit/874240eae3b66d8768666f72d036f752565c0700

**Comment by jpdillingham** [comment]
I couldn't figure it out either: #354

**Reply by mathiascode** [reply]
Thanks for the link, I missed that.
